sup3r/models/abstract.py

```diff
@@ -66,27 +67,6 @@ def seed(s=0):
         """
         CustomNetwork.seed(s=s)
 
-    @abstractmethod
-    def generate(self, low_res):
```

Comment:
From AbstractSingleModel? The linear models only inherit from AbstractInterface and so it could be worthwhile to keep the abstract generate method here.

Reply:
Oh yeah fair.